app/controllers/controllers.js

```diff
@@ -92,3 +92,15 @@ exports.patchArticleVotes = (req, res, next) => {
         next(err)
     })
 }
+
+exports.removeComment = (req, res, next) => { 
+    const commentId = req.params.comment_id
+
+    return deleteComment(commentId)
+    .then((result) => { 
+        res.status(200).send({deleted_comment : result[0]})
```

Comment:
better to just send the object back from the model here (not the rows array) to keep the controller clean in line with MVC convention